server/utils/nunjucksSetup.ts

```diff
@@ -177,6 +178,10 @@ export function registerNunjucks(app?: express.Express): Environment {
     return jsonDtToDateShort(dt)
   })
 
+  njkEnv.addFilter('datetimeToDateShortMinusDay', (dt: string) => {
```

Comment:
we should probably add examples of date formats in comments to each of these but it's out scope for this PR

Reply:
actually it'd probably be nicer to make filters more composable...

`myDate | minusDays(1) | jsonDtToDateShort`

rather than have lots of very specific filters

Reply:
(by the name of the filter I'd assumed it would show the date without the day)